time duration_time
string reco_type
float64 x_start
float64 y_start
float64 x_goal
float64 y_goal